StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/FormHeaderView.swift

```diff
@@ -39,6 +39,7 @@ final class FormHeaderView: UIView {
         let views = [imageView, label].compactMap { $0 }
         let stackView = UIStackView(arrangedSubviews: views)
         stackView.spacing = 12
+        stackView.alignment = .center
```

Comment:
Hmm, why is this needed?

Reply:
I believe the default is `.fill`, so the stackview's height ends up being the image's height, with the label shrinking to fit.